### PR TITLE
feat(doc): move install-from-source documentation to a separate file

### PR DESCRIPTION
**Pull request by kkovaacs**
This PR makes the following changes to our README:

- added some stuff in "Features"
- moved "install from source" section to a new document (tucked away in the doc directory)
- README now clearly suggests using Docker
- the example docker run command starts pathfinder in the backround with automatic restart (after reboot) enabled

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -263,73 +182,6 @@ Note that `chain-id` should be specified as text e.g. `SN_GOERLI`.
...
-
-# replace the value by of PATHFINDER_ETHEREUM_API_URL by the HTTP(s) URL pointing to your Ethereum node's endpoint
-cp example.pathfinder-var.env pathfinder-var.env
-
-docker-compose up -d
-```
-
-To check if it's running well use `docker-compose logs -f`.
-
 ## JSON-RPC API
```
Comment (CHr15F0x) [inline README.md:185]: Btw: there are instances of `v0.2.1-rc1` which should be changed to just `v0.2.1`

**Review by CHr15F0x — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -8,12 +8,15 @@ Pathfinder is currently in alpha so expect some rough edges but it is already us
 
 - access the full StarkNet state history
   - includes contract code and storage, and transactions
-- verifies state using L1
+- verifies state using L1 (Ethereum)
```
Comment (Mirko-von-Leipzig) [inline README.md:11]: Maybe just Ethereum
```suggestion
- verifies state using Ethereum
```

**Review by Mirko-von-Leipzig — APPROVED** [review]

**Comment by Mirko-von-Leipzig** [comment]
Should we also move our other doc stuff into `/doc`? RPC examples and specs..

**Review by Mirko-von-Leipzig — APPROVED** [review]

**Comment by kkovaacs** [comment]
> Should we also move our other doc stuff into `/doc`? RPC examples and specs..

I'm not quite sure `/doc` is the right place for the RPC examples. Can we maybe merge this as is?

**Comment by CHr15F0x** [comment]
>I'm not quite sure /doc is the right place for the RPC examples. Can we maybe merge this as is?

np, imo we can do more doc-refining in follow up PRs :+1:

**Pull request merged**